Update docs for flux-operator v0.6.0 (#91)
controlplaneio-fluxcd/distribution: docs/operator/fluxreport.md

The `.spec.reconcilers` field contains statistics about the Flux custom resources
that are reconciled by the Flux controllers, including the API version, kind, and
the number of resources in each state: failing, running and suspended.
For source type resources, the storage size of the locally cached artifacts is also reported.

Example:

```yaml
spec:
  reconcilers:
    - apiVersion: kustomize.toolkit.fluxcd.io/v1
      kind: Kustomization
      stats:
       failing: 1
       running: 5
       suspended: 5
    - apiVersion: source.toolkit.fluxcd.io/v1
      kind: GitRepository
      stats:
       failing: 1
       running: 2
       suspended: 3
       totalSize: 5.5 MiB
```

### Cluster sync status

The `.spec.sync` field contains information about the cluster sync status,
including the Flux Kustomization name, source URL, the applied revision,
and the sync readiness status.

Example:

```yaml
spec:
  sync:
    ready: true
    id: kustomization/flux-system
    path: tests/v2.3/sources
    source: https://github.com/controlplaneio-fluxcd/distribution.git
    status: 'Applied revision: refs/heads/main@sha1:a90cd1ac35de01c175f7199315d3f4cd60195911'
```

## Generating a FluxReport

> The FluxReport is automatically generated by the operator for the following conditions:

Reviewer: ```suggestion
The FluxReport is automatically generated by the operator after the following events:
```
